kubernetes/helm_charts/core/fusionauth/templates/ingress.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "fusionauth.fullname" . -}}
+{{- $ingressPaths := .Values.ingress.paths -}}
+{{- $extraPaths := .Values.ingress.extraPaths -}}
+apiVersion: {{ include "fusionauth.ingressApiVersion" . }}
+kind: Ingress
```

Comment:
We dont have Ingress in our Kube cluster

Reply:
Removed ingress.yaml and related references.